### PR TITLE
Fixing Jaguar tests to not test values that are not present in old regression tests.

### PR DESCRIPTION
**Pull request by shivupa**
Addresses all the jaguar regression test errors in #673

### CODE REVIEW

**Comment by berquist** [comment]
Confirmation that these fixes work using a regression output diff:
```diff
477,565c477,479
<   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar4.2/dvb_sp.out ... 4 test(s) had errors
< Error for testgeotargets (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 168, in testgeotargets
<     dim_geotargets = self.data.geotargets.shape
< AttributeError: 'ccData' object has no attribute 'geotargets'
< 
< Error for testgeovalues_atomcoords (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 151, in testgeovalues_atomcoords
<     count_geovalues = len(self.data.geovalues)
< AttributeError: 'ccData' object has no attribute 'geovalues'
< 
< Error for testgeovalues_scfvalues (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 161, in testgeovalues_scfvalues
<     count_geovalues = len(self.data.geovalues)
< AttributeError: 'ccData' object has no attribute 'geovalues'
< 
< Error for testoptdone (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 176, in testoptdone
<     self.assertTrue(self.data.optdone)
< AttributeError: 'ccData' object has no attribute 'optdone'
< 
<   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar4.2/dvb_sp_b.out ... 4 test(s) had errors
< Error for testgeotargets (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 168, in testgeotargets
<     dim_geotargets = self.data.geotargets.shape
< AttributeError: 'ccData' object has no attribute 'geotargets'
< 
< Error for testgeovalues_atomcoords (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 151, in testgeovalues_atomcoords
<     count_geovalues = len(self.data.geovalues)
< AttributeError: 'ccData' object has no attribute 'geovalues'
< 
< Error for testgeovalues_scfvalues (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 161, in testgeovalues_scfvalues
<     count_geovalues = len(self.data.geovalues)
< AttributeError: 'ccData' object has no attribute 'geovalues'
< 
< Error for testoptdone (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 176, in testoptdone
<     self.assertTrue(self.data.optdone)
< AttributeError: 'ccData' object has no attribute 'optdone'
< 
<   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar4.2/dvb_un_sp.out ... 1 test(s) had errors
< Error for testmosyms (__main__.JaguarSPunTest_nmo_all)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/testSPun.py", line 197, in testmosyms
<     shape0 = (len(self.data.mosyms), len(self.data.mosyms[0]))
< AttributeError: 'ccData' object has no attribute 'mosyms'
< 
---
>   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar4.2/dvb_sp.out ... parsed and tested
>   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar4.2/dvb_sp_b.out ... parsed and tested
>   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar4.2/dvb_un_sp.out ... parsed and tested
567,587c481,482
<   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar6.0/dvb_sp.out ... 1 test(s) had errors
< Error for testsymlabels (__main__.JaguarSPTest_6_31gss)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   [Previous line repeated 1 more time]
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 144, in testsymlabels
<     sumwronglabels = sum([x not in ['Ag', 'Bu', 'Au', 'Bg'] for x in self.data.mosyms[0]])
< AttributeError: 'ccData' object has no attribute 'mosyms'
< 
<   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar6.0/dvb_un_sp.out ... 1 test(s) had errors
< Error for testmosyms (__main__.JaguarSPunTest_nmo_all)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/testSPun.py", line 197, in testmosyms
<     shape0 = (len(self.data.mosyms), len(self.data.mosyms[0]))
< AttributeError: 'ccData' object has no attribute 'mosyms'
< 
---
>   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar6.0/dvb_sp.out ... parsed and tested
>   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar6.0/dvb_un_sp.out ... parsed and tested
590,637c485,486
<   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar6.5/dvb_sp.out ... 4 test(s) had errors
< Error for testgeotargets (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 168, in testgeotargets
<     dim_geotargets = self.data.geotargets.shape
< AttributeError: 'ccData' object has no attribute 'geotargets'
< 
< Error for testgeovalues_atomcoords (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 151, in testgeovalues_atomcoords
<     count_geovalues = len(self.data.geovalues)
< AttributeError: 'ccData' object has no attribute 'geovalues'
< 
< Error for testgeovalues_scfvalues (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 161, in testgeovalues_scfvalues
<     count_geovalues = len(self.data.geovalues)
< AttributeError: 'ccData' object has no attribute 'geovalues'
< 
< Error for testoptdone (__main__.JaguarGeoOptTest_nmo45)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 176, in testoptdone
<     self.assertTrue(self.data.optdone)
< AttributeError: 'ccData' object has no attribute 'optdone'
< 
<   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar6.5/dvb_un_sp.out ... 1 test(s) had errors
< Error for testmosyms (test.data.testSPun.JaguarSPunTest)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/testSPun.py", line 197, in testmosyms
<     shape0 = (len(self.data.mosyms), len(self.data.mosyms[0]))
< AttributeError: 'ccData' object has no attribute 'mosyms'
< 
---
>   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar6.5/dvb_sp.out ... parsed and tested
>   /home/eric/development/cclib_berquist/test/../data/regression/Jaguar/Jaguar6.5/dvb_un_sp.out ... parsed and tested
825c674
< Total: 437   Failed: 0  Errors: 35
---
> Total: 437   Failed: 0  Errors: 19
```

File: test/regression.py [hunk 1]
```diff
@@ -2069,18 +2101,18 @@ def testatommasses(self):
 
     "Jaguar/Jaguar4.2/dvb_gopt.out":    JaguarGeoOptTest_nmo45,
     "Jaguar/Jaguar4.2/dvb_gopt_b.out":  GenericGeoOptTest,
-    "Jaguar/Jaguar4.2/dvb_sp.out":      JaguarGeoOptTest_nmo45,
-    "Jaguar/Jaguar4.2/dvb_sp_b.out":    JaguarGeoOptTest_nmo45,
-    "Jaguar/Jaguar4.2/dvb_un_sp.out":   JaguarSPunTest_nmo_all,
+    "Jaguar/Jaguar4.2/dvb_sp.out":      JaguarGeoOptTest_nmo45_nogeo,
```
Comment (langner) [inline test/regression.py]: It's unclear to me why we would expect geometry optimization for an SP job anyway. Can you try use JaguarSPTest its variant here?

File: test/regression.py [hunk 2]
```diff
@@ -2069,18 +2101,18 @@ def testatommasses(self):
 
     "Jaguar/Jaguar4.2/dvb_gopt.out":    JaguarGeoOptTest_nmo45,
     "Jaguar/Jaguar4.2/dvb_gopt_b.out":  GenericGeoOptTest,
-    "Jaguar/Jaguar4.2/dvb_sp.out":      JaguarGeoOptTest_nmo45,
-    "Jaguar/Jaguar4.2/dvb_sp_b.out":    JaguarGeoOptTest_nmo45,
-    "Jaguar/Jaguar4.2/dvb_un_sp.out":   JaguarSPunTest_nmo_all,
+    "Jaguar/Jaguar4.2/dvb_sp.out":      JaguarGeoOptTest_nmo45_nogeo,
+    "Jaguar/Jaguar4.2/dvb_sp_b.out":    JaguarGeoOptTest_nmo45_nogeo,
```
Comment (langner) [inline test/regression.py]: Ditto.

**Comment by berquist** [comment]
Can you change the base branch to `673-fix-silent-regression-failures`?

**Comment by shivupa** [comment]
This is updated and should be good to go!

**Comment by langner** [comment]
Great, thanks. I confirmed in Travis that Jaguar regression now have no errors.

**Pull request merged**